coinstacks/bitcoin/api/src/app.ts

```diff
@@ -42,4 +47,4 @@ app.get('/', async (_, res) => {
 app.use(middleware.errorHandler)
 app.use(middleware.notFoundHandler)
 
-app.listen(port, () => logger.info('server listening...'))
+app.listen(port, () => logger.info({ port }, 'Server started'))
```

Comment:
I had removed the port from the log as it was causing confusing during local dev, just fyi